support-frontend/assets/pages/[countryGroupId]/guardianLightLanding/components/guardianLightCard.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+	from,
+	headlineBold17,
+	headlineBold24,
+	palette,
+	space,
+	until,
+} from '@guardian/source/foundations';
+import { LinkButton, themeLinkBrand } from '@guardian/source/react-components';
+import { Divider } from '@guardian/source-development-kitchen/react-components';
+import { BenefitsCheckList } from 'components/checkoutBenefits/benefitsCheckList';
+import { type ProductDescription } from 'helpers/productCatalog';
+
+export type CardPosition = 1 | 2;
+export type GuardianLightCardProps = {
+	cardPosition: CardPosition;
+	link: string;
+	productDescription: ProductDescription;
+	ctaCopy: string;
+};
+
+const container = css`
+	position: 'static';
+	background-color: ${palette.neutral[100]};
+	border-radius: ${space[3]}px;
+	padding: ${space[6]}px ${space[5]}px ${space[6]}px ${space[5]}px;
+	${until.desktop} {
+		order: ${2};
+		padding: ${space[4]}px ${space[3]}px ${space[4]}px ${space[3]}px;
+		margin-top: ${'0'}px;
+	}
+`;
+const titleSummarySvgCss = css`
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	align-items: center;
+	margin-bottom: ${`${space[4]}px`};
+	${from.desktop} {
+		margin-bottom: ${space[6]}px;
+	}
+`;
+const svgCss = css`
+	margin-bottom: ${`${space[2]}px`};
+	${from.desktop} {
+		margin-bottom: ${space[4]}px;
+	}
+`;
+const titleCss = css`
+	text-align: center;
+	${headlineBold17};
+	color: ${palette.brand[100]};
+	${from.desktop} {
+		${headlineBold24};
+	}
+`;
+const btnStyleOverrides = css`
+	width: 100%;
+	justify-content: center;
+	margin-bottom: ${space[1]}px;
+`;
+const dividerCss = css`
+	width: 100%;
+	margin: ${space[4]}px 0;
+	${from.desktop} {
+		margin: ${space[6]}px 0;
+	}
+`;
+const checkmarkBenefitList = css`
+	text-align: left;
+	width: 100%;
+	${from.desktop} {
+		width: 90%;
+	}
+`;
+
+export function GuardianLightCard({
+	cardPosition,
+	link,
+	productDescription,
+	ctaCopy,
+}: GuardianLightCardProps): JSX.Element {
+	const quantumMetricButtonRef = `guardianLight-${cardPosition}-button`;
+	const { label, benefits } = productDescription;
+	const icon = cardPosition === 1 ? leftSvgIcon : rightSvgIcon;
+	return (
+		<section css={container}>
+			<div css={titleSummarySvgCss}>
+				<div css={svgCss}>{icon}</div>
+				<h2 css={titleCss}>{label}</h2>
+			</div>
+			<ThemeProvider theme={themeLinkBrand}>
+				<LinkButton
+					href={link}
+					cssOverrides={btnStyleOverrides}
+					data-qm-trackable={quantumMetricButtonRef}
+				>
+					{ctaCopy}
+				</LinkButton>
+			</ThemeProvider>
+			<Divider cssOverrides={dividerCss} />
+			<BenefitsCheckList
+				benefitsCheckListData={benefits.map((benefit) => {
+					return {
+						text: benefit.copy,
+						isChecked: true,
+						toolTip: benefit.tooltip,
+						isNew: benefit.isNew,
+						hideBullet: benefit.hideBullet,
+					};
+				})}
+				style={'bullet'}
+				iconColor={palette.brand[500]}
+				cssOverrides={checkmarkBenefitList}
+			/>
+		</section>
+	);
+}
+
+const leftSvgIcon = (
```

Comment:
I think `leftSvgIcon` and `rightSvgIcon` could be extracted from this file into their own files, they may not be used elsewhere yet but there's a lot of `svg` mark-up here to scroll through, that we're unlikely to ever edit?

Reply:
Yep, unlikely to edit, have moved these out now. Thanks.